### ISSUE TITLE
Functionality

### ISSUE THREAD

**Issue by zschrunk** [Firefly9093/web-iv-portfolio-site#4]
The images don't have a hover state. The Facebook link isn't working, it tries to take me to a page but it gives me a warning. The work link in your nav takes me to a project instead of the work section on the home page. That can get confusing and also make people think that you only have one project. The send button isn't reading as a button so it stays as an arrow instead of switching to a hand when hovering over it.

Below Satisfactory

**Comment by e-m-durham** [comment]
I agree @zschrunk on these, while some of them overlap UX. Also, since you still have pieces I assume you're still developing, like the navigation, it's tough to anticipate the evaluation of functionality in the final product.
